src/couchdb/rolling-database.ts

```diff
@@ -354,7 +354,11 @@ export function makeRollingDatabase<T>(
       cleanups = []
       for (const { archived, name } of list) {
         const setup: DatabaseSetup = { name, ...setupRest }
-        if (!archived) setup.replicatorSetup = replicatorSetup
+        if (archived) {
+          setup.ignoreMissing = true
+        } else {
+          setup.replicatorSetup = replicatorSetup
+        }
```

Comment:
This is like a sort of "soft-create" idea? Don't create the database, but rather just make sure it's setup correctly?

Reply:
That's the idea. We can delete archived databases from the production system, and they won't come back, but on our backup boxes, we will continue to maintain whatever index documents and replications we need.